src/Interfaces/TransfersInterfaces/index.ts

```diff
@@ -65,10 +65,8 @@ export type IncludedQueryParams = {
       }
     }
     fee: {
-      select: {
-        amount: boolean
```

Comment:
Why don't we need this info anymore?

Reply:
We do need it. But prisma doesn't allow us to use select and include together, so now we return all fields from fee collection + all fields from resource collection